public/src/api/httpClient.ts

```diff
@@ -70,6 +70,17 @@ export function getUser(
   });
 }
 
+export function getUserData(
+  email: string,
+  config: AxiosRequestConfig = {}
+): Promise<any> {
+  return makeApiRequest({
+    url: `${ApiUrl.Users}/one/${email}`,
```

Comment:
Use `trim` if you pass `email` as part of the path
```suggestion
    url: `${ApiUrl.Users}/one/${email.trim()}`,
```